api/custom_index.py

```diff
@@ -10,6 +10,7 @@
 from sqlalchemy.orm.session import Session
 
 from config import CannotLoadConfiguration
+from core.app_server import cdn_url_for
```

Comment:
This missing import didn't cause tests to fail because the tests pass in a mock `url_for`.